examples/ios/HelloWorldSwift/Info.plist

```diff
@@ -2,6 +2,11 @@
 <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
 <plist version="1.0">
 <dict>
+	<key>UILaunchScreen</key>
+	<dict>
+		<key>UILaunchScreen</key>
+		<dict/>
+	</dict>
```

Comment:
This more compact form still works for me, mind updating it?
```suggestion
	<key>UILaunchScreen</key>
	<dict/>
```

Reply:
Haven't reverse-engineered the code detects "UILaunchScreen" here, and this key-value was found and extracted by the generated binary plist file of Xcode's SwiftUI Demo App.

I would recommend that we stick to what Xcode generates in its Info.plist to maximize compatibility.

But if the compact format is working now. We can also update it if you like.

Reply:
Confirmed! I hadn't realized you'd checked that, thanks for being so thorough.

Xcode 15.2 with SwiftUI + Swift iOS app gives this in the generated `.app/Info.plist`:
```XML
	<key>UILaunchScreen</key>
	<dict>
		<key>UILaunchScreen</key>
		<dict/>
	</dict>
```